src/_utils.js

```diff
@@ -31,10 +31,20 @@ export const addClassName = (path, jsxId) => {
     const node = attr.node
     if (t.isJSXSpreadAttribute(attr)) {
       const name = node.argument.name
-      const attrNameDotClassName = t.memberExpression(
-        t.identifier(name),
-        t.identifier('className')
-      )
+      let attrNameDotClassName
+
+      if (t.isMemberExpression(node.argument)) {
+        attrNameDotClassName = t.memberExpression(
+          node.argument,
+          t.identifier('className')
+        )
+      } else {
+        attrNameDotClassName = t.memberExpression(
+          t.identifier(name),
+          t.identifier('className')
+        )
+      }
```

Comment:
nitpick: I just realized that this could be:

```js
const attrNameDotClassName = t.memberExpression(
  t.isMemberExpression(node.argument) ? node.argument : t.identifier(name),
  t.identifier('className')
)
```